W-13932478 retest main tuts duke (#119)
mulesoft/docs-code-builder: modules/ROOT/pages/tut-design-api-specification-from-scratch.adoc

image::acb-api-project.png["Project for American Airlines API"]

. In the activity bar of the IDE, click the *Anypoint Code Builder* (MuleSoft) icon to navigate to Anypoint Code Builder.
+
For more guidance, see xref:design-api-specification.adoc#start-spec[Start an API Specification].
. From the *Quick Actions* menu, click *Design an API* from the Anypoint Code Builder menu to start the API specification project:
+
[%header,cols="1a,1a"] 
|===
| Field Name | Field Value

| *Project Name* | *American Flights API* or a similar name of your choice
| *Project Location* | Your home directory or another directory you create
| *API Specification Language* | *RAML 1.0*
| *Business Group* | Group for your API specification
|===
. Select *Create Project* to generate the American Flights API project file: `american-flights-api.raml`.
+
The file name is based on the name you provide for the project.
. Proceed to <<open-project>>.

[[open-project]]

> == Open the Workspace for Your API Specification Project

Reviewer: These next four steps are unnecessary on Desktop, because the project's workspace is automatically opened after selecting "create project". Does this behavior differ on web? @dukesphere

Author: oh good! I used to get an issue where if I created a folder to put the project in from the *Project Location* field, I had to navigate to the project folder. Maybe it was fixed or ¯\_(ツ)_/¯ . Removing the section. Thx!

Author: @chris-campbell9 Confirmed and removed section. Thx 😄

+
image::acb-api-console.png["A specification with get and post methods"]
+

> Wait for the API Console to load. 

Reviewer: On Desktop, I had to click back in the editor window to get the API Console to display my API. If I didn't click in the editor, the API Console was present but empty, which was confusing. Does this behavior differ from web? Or perhaps it's a product bug? I can pursue this with eng, but I thought I'd check to see if you knew offhand. @dukesphere

Author: Ohhh, that happens to me sometimes but I didn't connect the dots. Seems like a bug to me, I/we can tell them to click in the edit window for now. Thx!

Author: I thought it was a loading delay or something.

> Wait for the API Console to load. 

Author: ```suggestion
Click in the RAML file you are editing, and wait for the API Console to load. 
```

> . Open the API Console to display the nested resource:

Reviewer: ```suggestion
. View the nested resource in the API Console:
```
The API Console was already open due to following the steps in the previous section. Not a big deal, but I'm raising a question because I wonder if the customer could close the console between sections? If so, then disregard my suggestion. @dukesphere
